app/assets/stylesheets/moderators.scss

```diff
@@ -385,8 +388,103 @@
   .reaction-vomit-button.reacted .reacted-emoji {
     display: block;
   }
+  .other-things-container {
+    border-top: 1px solid var(--base-20);
+    height: 100%;
+    button.other-things-btn {
+      background: inherit;
+      border: none;
+      padding: $su-3 $su-4;
+      width: 100%;
+      display: flex;
+      &:hover {
+        background-color: var(--base-inverted);
+        .label-wrapper > .icon {
+          background-color: var(--accent-brand);
+          svg {
+            fill: var(--base-inverted);
+          }
+        }
+        header h2 {
```

Comment:
Hey Andy, it seems here you set the headings 'Adjust tags' and 'Set experience level' to change color to `accent-brand` on hover; however, I am not seeing this behavior in the browser (see video below)

https://www.loom.com/share/f8cf677b554f4fb780c4a70cacd1fc53

Reply:
Ahh good catch, thanks!